### PR TITLE
Update version numbers

### PR DESCRIPTION
**Pull request by KevinRansom**
FSharp.Core 4.7.2 is now published, time to prepare 4.7.3

Update tooling version numbers to prepare to 4.9.2

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -14,17 +14,17 @@
...
     <!-- The current published nuget package -->
-    <FSharpCoreShippedPackageVersion>4.7.1</FSharpCoreShippedPackageVersion>
+    <FSharpCoreShippedPackageVersion>4.7.2</FSharpCoreShippedPackageVersion>
     <!-- The pattern for specifying the preview package -->
     <FSharpCorePreviewPackageVersion>$(FSCorePackageVersion)-$(PreReleaseVersionLabel).*</FSharpCorePreviewPackageVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <FSPackageMajorVersion>10.8</FSPackageMajorVersion>
-    <FSPackageVersion>$(FSPackageMajorVersion).1</FSPackageVersion>
+    <FSPackageMajorVersion>10.9</FSPackageMajorVersion>
```
Comment (brettfo) [inline eng/Versions.props:26]: This shouldn't be in this pr, I'll come in with the merge from 16.6

Comment (KevinRansom) [reply eng/Versions.props:26]: kewl

Comment (KevinRansom) [reply eng/Versions.props:26]: @brettfo , done.

File: eng/Versions.props [hunk 2]
```diff
@@ -14,17 +14,17 @@
...
     <!-- The current published nuget package -->
-    <FSharpCoreShippedPackageVersion>4.7.1</FSharpCoreShippedPackageVersion>
+    <FSharpCoreShippedPackageVersion>4.7.2</FSharpCoreShippedPackageVersion>
     <!-- The pattern for specifying the preview package -->
     <FSharpCorePreviewPackageVersion>$(FSCorePackageVersion)-$(PreReleaseVersionLabel).*</FSharpCorePreviewPackageVersion>
   </PropertyGroup>
   <PropertyGroup>
     <FSPackageMajorVersion>10.9</FSPackageMajorVersion>
-    <FSPackageVersion>$(FSPackageMajorVersion).1</FSPackageVersion>
+    <FSPackageVersion>$(FSPackageMajorVersion).2</FSPackageVersion>
```
Comment (brettfo) [inline eng/Versions.props:27]: `10.9.1` should probably change to `10.10.0` instead of `10.9.2`.

Comment (KevinRansom) [reply eng/Versions.props:27]: @Brettfo, done.

**Pull request merged**